app/(route)/mypage/gallery/_artwork-box.tsx

```
@@ -0,0 +1,43 @@
+  const handleClose = () => {
+    setArtworkNumber(null);
+  };
+
+  return (
+    <>
+      <div className="overflow-x-hidden absolute bottom-[235px] left-0 w-full z-10">
+        <div className="w-full h-full overflow-hidden">
+          <div className="w-full h-full overflow-x-auto">
+            <div className="min-w-max flex flex-row items-end h-[450px]">
+              {artworks.map((artwork, index) => (
+                <div key={artwork.artworkUid} className="relative pl-[100px] [&:last-child]:pr-[100px] cursor-pointer" onClick={() => handleOpen(index)}>
```

Biome (1.9.4)

[error] Enforce to have the onClick mouse event with the onKeyUp, the onKeyDown, or the onKeyPress keyboard event.

Actions triggered using mouse events should have corresponding keyboard events to account for keyboard-only navigation.

(lint/a11y/useKeyWithClickEvents)